js/main.js

```diff
@@ -28,8 +28,17 @@ const selectRandom = (memberImgArr, n) => {
   return result;
 };
 
+const numOfMembers = 5;
+let memberSection = document.getElementById('members');
+for (let i = 0; i < numOfMembers; i++) {
+  const memberTags = document.createElement('img');
+  memberTags.classList.add('member-img', 'member_animation');
+  memberTags.setAttribute('alt', 'member-img');
```

Comment:
`Alt` text could non-technical? It should be simple english, if the images fail to load, hovering upon them we should see that text right?